cljfmt/src/cljfmt/core.cljc

```diff
@@ -568,10 +568,43 @@
 #?(:clj
    (defn- as-zloc->alias-mapping [as-zloc]
      (let [alias             (some-> as-zloc z/right z/sexpr)
-           current-namespace (some-> as-zloc z/leftmost z/sexpr)
+           ;; `leftmost` might be an `uneval` node e.g. in
```

Comment:
I tried to write out the reasoning behind the changes here as opposed to in the PR, hope that's ok. Maybe it will save the next person who comes along some time so they don't have to work out what's going on the hard way